src/content/enterprise/administration/configuration.mdx

```diff
@@ -226,6 +226,7 @@ It is possible to use separate cloud providers for DNS than for storage if neede
 The API service. Account and Kubernetes credentials management, namespace creation, and sharing, deployment via the catalog, etc...
 
 - `annotations`: Annotations to add to the API pods.
+- `env`: Environment variables to add to the API pods.
```

Comment:
I think we should follow the pattern that https://artifacthub.io/packages/helm/bitnami/mysql or [gitlab](https://artifacthub.io/packages/helm/gitlab/gitlab?modal=values&path=global.extraEnv) uses and call these `extraEnvVars` or ` extraEnv` so it's more obvious that this will add to the existing (whatever this might be) instead of replacing it entirely (which is what I'd think when I see a property called `env`).

Reply:
I agree. Let me see if we can change that before 0.14 is released